clients/oxide-client/src/lib.rs

```diff
@@ -7,10 +7,11 @@
 use anyhow::Context;
 use anyhow::anyhow;
 use futures::FutureExt;
-use hickory_resolver::TokioAsyncResolver;
+use hickory_resolver::TokioResolver;
```

Comment:
ah, i see, the synchronous Hickory resolver is just _gone_ gone (PR 2515 over there) and names shifted accordingly. good for them!